docs/testing/troubleshooting.md

```diff
@@ -1,31 +1,65 @@
 ---
 title: 'Troubleshooting Office Scripts'
 description: 'Debugging tips and techniques for Office Scripts, as well as help resources.'
-ms.date: 12/16/2020
+ms.date: 05/10/2021
 localization_priority: Normal
 ---
 
 # Troubleshooting Office Scripts
 
-As you develop Office Scripts, you may make mistakes. It's okay. We have tools that help find the problems and get your scripts working perfectly.
+As you develop Office Scripts, you may make mistakes. It's okay. You have the tools to help find the problems and get your scripts working perfectly.
+
+## Types of errors
+
+Office Scripts errors fall into one of two categories:
+
+1. Compile-time errors or warnings.
+1. Runtime error.
+
+### Compile-time errors
+
+Compile-time errors and warnings are initially shown in the Code Editor. These are shown by the wavy red underlines in the editor. They are also displayed under the **Problems** tab at the bottom of the Code Editor task pane. Selecting the error will give more details about the problem and suggest solutions. Compile-time errors should be addressed before running the script.
+
+:::image type="content" source="../images/explicit-any-editor-message.png" alt-text="A compiler error shown in the Code Editor's hover text":::
+
+You may also see orange warning underlines and grey informational messages. These indicate performance suggestions or other possibilities where the script may have unintentional effects. Such warnings should be examined closely, though they may not actually be problems.
```

Comment:
How about... 
```suggestion
You may also see orange warning underlines and grey informational messages. These indicate performance suggestions or other possibilities where the script may have unintentional effects. Such warnings should be examined closely, though these issues may not prevent your script from running properly.
```

Reply:
Hmm. I'm trying to say "the warning might be wrong", without making the system seem unreliable.